packages/ia-components/components/theatres/with-youtube-spotify/utils/archive-default-album-parser.js

```diff
@@ -39,10 +39,11 @@ const archiveDefaultAlbumParser = ({ fileDirectoryPrefix, files }) => {
     const isOriginal = source === 'original';
     const isAudioFile = isValidAudioFile(currentFileName);
     const isItemImageFile = isOriginal && isValidImageFile(currentFileName);
-
+    const isThumbnail = !!currentFileName.match(/__ia_thumb.jpg/g);
```

Comment:
Hi Isa! Would it make sense to anchor this regex to the end of the string to make it a bit more strict, eg `/__ia_thumb.jpg$/g`?

Reply:
hi matt, thanks for the suggestion. updated.